tests/spec_test.py

```diff
@@ -1,4 +1,4 @@
-"""Tests for core spec objects."""
+'''Tests for core spec objects.'''
```

Comment:
No, good catch. It was a find/replace error. I'll search for these and push an update.